e2e-tests/Dockerfile

```diff
@@ -36,8 +36,8 @@ RUN curl -fsSLo /tmp/geckodriver.tar.gz https://github.com/mozilla/geckodriver/r
   && ln -fs /opt/geckodriver-$GECKODRIVER_VERSION /usr/bin/geckodriver
 
 WORKDIR /src
-COPY Pipfile /src/
-RUN pipenv install --system --skip-lock
+COPY Pipfile* /src/
+RUN pipenv install --system --deploy --ignore-pipfile
```

Comment:
I don't think we need `--ignore-pipfile` as the Pipfile.lock will be used by default, and with `--deploy` we'll abort if the lock file is out of date.

Reply:
Thanks; fixed.